pkg/cmd/scan/output/console.go

```diff
@@ -50,22 +49,20 @@ func (c *Console) Write(analysis *analyser.Analysis) error {
 		for ty, resource := range unmanagedByType {
 			fmt.Printf("  %s:\n", ty)
 			for _, res := range resource {
-				stringer, ok := res.(fmt.Stringer)
-				fmt.Printf("    - %s", res.TerraformId())
-				if ok {
-					fmt.Printf(" (%s)", stringer.String())
+				humanString := res.TerraformId()
+				if stringer, ok := res.(fmt.Stringer); ok {
+					humanString = stringer.String()
 				}
-				fmt.Println()
+				fmt.Printf("    - %s\n", humanString)
 			}
 		}
 	}
 
 	if analysis.Summary().TotalDrifted > 0 {
 		fmt.Printf("Found drifted resources:\n")
 		for _, difference := range analysis.Differences() {
-			stringer, ok := difference.Res.(fmt.Stringer)
 			humanString := difference.Res.TerraformType()
-			if ok {
+			if stringer, ok := difference.Res.(fmt.Stringer); ok {
```

Comment:
This diff does not change anything. Here we want to make the diff output identical as when resource is unmanaged or deleted.

```
stringer, ok := difference.Res.(fmt.Stringer)
humanString := difference.Res.TerraformType()
if ok {
  humanString = stringer.String()
}
fmt.Printf("  - %s (%s):\n", difference.Res.TerraformId(), humanString)
```

Should became something like this

```
stringer, ok := difference.Res.(fmt.Stringer)
humanString := difference.Res.TerraformId()
if ok {
  humanString = stringer.String()
}
fmt.Printf("  - %s (%s):\n", humanString, difference.Res.TerraformType())
```

Reply:
Yes i saw your comment on this i was working on it, i push commit soon.